artemis-protocols/artemis-amqp-protocol/src/main/java/org/apache/activemq/artemis/protocol/amqp/connect/mirror/AMQPMirrorControllerSource.java

```diff
@@ -162,6 +172,12 @@ public void createQueue(QueueConfiguration queueConfiguration) throws Exception
          }
          return;
       }
+      if (ignoreAddress(queueConfiguration.getAddress())) {
+         if (logger.isTraceEnabled()) {
+            logger.trace("Skipping create " + queueConfiguration + ", queue address" + queueConfiguration.getAddress() + " is excluded");
```

Comment:
Given the filter does actually have 'exclusions' and it might not actually be excluded, so much as just not-included, I would reword this to avoid saying it is 'excluded' specifically, but instead something more general like 'doesnt match filter'.